src/Picker/BynderAssetPickerProvider.php

```diff
@@ -50,7 +50,7 @@ public function __construct(FactoryInterface $menuFactory, RouterInterface $rout
      *
```

Comment:
Looks like CS fixer would help here to remove all that additional phpdoc clutter ;)